calcom/cal.com
# Use the matched user email to send the password reset to

## What does this PR do?

* Minor cleanup
* Implements toLowerCase()
* Prevents misuse of lowercasing by using the matched user email.

## Type of change


## How to test
Go to login page, reset password using any capitalisation for your email. It works. TODO maybe add an end-to-end test to ensure the email goes out and all that.

## Checklist:
